.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Steps to reproduce
+
+### Expected result
```

Comment:
I would add 
### Live demo
with a link to jsbin/jsfiddle, we should prepare an example that uses polygit to load juicy components to be used for forking, as in https://github.com/Polymer/polymer/blob/master/.github/ISSUE_TEMPLATE.md#live-demo